src/index.ts

```diff
@@ -2521,6 +2521,765 @@ export enum ButtonVariant {
+  text: TextThemeConfig
+  link: TextThemeConfig
+}
+
+export interface PreferenceWelcomeTabThemeConfig {
+  container: WelcomeTabContainerThemeConfig
+  header: WelcomeTabHeaderThemeConfig
+  navigation: WelcomeTabNavigationThemeConfig
+  exitButton: WelcomeTabExitButtonThemeConfig
+  heroBanner: WelcomeTabHeroBannerThemeConfig
+  welcomeMsg: WelcomeTabWelcomeMsgThemeConfig
+  quickLinks: WelcomeTabQuickLinksThemeConfig
+  about: WelcomeTabAboutThemeConfig
+}
+
+/** Preferences - Privacy Policy Theme Config */
+export interface PreferencePrivacyPolicyTabThemeConfig {
+  text: TextThemeConfig
+  link: TextThemeConfig
+}
+
+/** Preferences - Purposes Theme Config */
+
+export interface PurposesTabHeaderThemeConfig {
+  title: TextThemeConfig
+  description: TextThemeConfig
+  link: TextThemeConfig
+}
+
+export interface PreferencePurposesTabThemeConfig {
+  header: PageTextThemeConfig
+  listHeader: PurposeListHeaderThemeConfig
+  list: PurposeListThemeConfig
+  footer: FooterThemeConfig
+}
+
+/** Preferences - Subscriptions Theme Config */
+
+export enum SubscriptionListStyle {
+  Filled = 'filled',
+  Underlined = 'underlined',
+}
+
+export interface SubscriptionsTabGlobalControlThemeConfig {
+  background: ColorThemeConfig
+  text: ColorThemeConfig
+  cornerRadius: number
+  switchButton: SwitchButtonsThemeConfig
+}
+
+export interface LayoutThemeConfig {
+  style: SubscriptionListStyle
+  background: ColorThemeConfig
+  text: TextThemeConfig
+  link: TextThemeConfig
+  cornerRadius: number
+}
+
+export interface SubscriptionsTabListThemeConfig {
+  layout: LayoutThemeConfig
+  switchButton: SwitchButtonsThemeConfig
+  checkbox: CheckboxesThemeConfig
+}
+
+export interface PreferenceSubscriptionsTabThemeConfig {
+  header: PageTextThemeConfig
+  globalControl: SubscriptionsTabGlobalControlThemeConfig
+  list: SubscriptionsTabListThemeConfig
+  footer: FooterThemeConfig
+}
+
+/** Preferences - Requests Theme Config */
+
+export enum FormFieldStyle {
+  Box = 'box',
+  Underline = 'underline'
+}
+
+export interface RequestsTabDsrLinkThemeConfig {
+  title: TextThemeConfig
+  description: TextThemeConfig
+  arrowIcon: ColorThemeConfig
+  background: ColorThemeConfig
+  cornerRadius: number
+}
+
+export interface RightsListItemThemeConfig {
+  title: TextThemeConfig
+  subtitle: TextThemeConfig
+  arrowIcon: ColorThemeConfig
+  background: ColorThemeConfig
+  cornerRadius: number
+}
+
+export interface RequestsTabRightsListThemeConfig {
+  title: TextThemeConfig
+  item: RightsListItemThemeConfig
+}
+
+export interface RequestsTabHomeThemeConfig {
+  header: PageTextThemeConfig
+  dsrPortalLink: RequestsTabDsrLinkThemeConfig
+  rightsList: RequestsTabRightsListThemeConfig
+}
+
+export interface FormDividersThemeConfig {
+  title: TextThemeConfig
+  subtitle: TextThemeConfig
+}
+
+export interface FieldThemeConfig {
+  background: ColorThemeConfig
+  outline: ColorThemeConfig
+}
+
+export interface FormFieldsThemeConfig {
+  style: FormFieldStyle
```

Comment:
talk

Reply:
For Misha meeting - What does underline mean for a "form field style"?